File: app/src/main/java/com/botijasoftware/utils/renderer/RenderItem.java
package com.botijasoftware.utils.renderer;

import com.botijasoftware.utils.GLMatrix;
import com.botijasoftware.utils.IndexBuffer;
import com.botijasoftware.utils.VertexBuffer;
import com.botijasoftware.utils.materials.Material;

public class RenderItem {
	public VertexBuffer vb;
	public Material material;
	public IndexBuffer ib;
	public GLMatrix mvmatrix = new GLMatrix();
	
	public RenderItem(VertexBuffer vb, IndexBuffer ib, GLMatrix mvmatrix, Material material) {
		this.vb = vb;
		this.ib = ib;
		this.mvmatrix.restore(mvmatrix);
		this.material = material;
	}
}

File: app/src/main/java/com/botijasoftware/utils/renderer/RenderItem.kt
package com.botijasoftware.utils.renderer

import com.botijasoftware.utils.GLMatrix
import com.botijasoftware.utils.IndexBuffer
import com.botijasoftware.utils.VertexBuffer
import com.botijasoftware.utils.materials.Material

class RenderItem(var vb: VertexBuffer, var ib: IndexBuffer, mvmatrix: GLMatrix, var material: Material) {
    var mvmatrix = GLMatrix()

    init {
        this.mvmatrix.restore(mvmatrix)
    }
}